### ISSUE TITLE
onReady callback never called when text have less lines than `numberOfLines`

### ISSUE THREAD

**Issue by matziol** [fawaz-ahmed/react-native-read-more#68]
When passed text is shorter and evaluating to less lines than `numberOfLines`. This is how it looks with example app and `numberOfLines={3}`:
```
<ReadMore
  numberOfLines={3}
  style={styles.textStyle}
  onReady={() => Alert.alert('onReady Called')}>
  {
    "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum."
  }
</ReadMore>
``` 

https://user-images.githubusercontent.com/73690392/176875170-faeb27b6-523c-41fb-8284-f0d19eb98d95.mp4

AND

```
<ReadMore
  numberOfLines={3}
  style={styles.textStyle}
  onReady={() => Alert.alert('onReady Called')}>
  {'Lorem Ipsum is simply dummy text'}
</ReadMore>
``` 

https://user-images.githubusercontent.com/73690392/176875337-d5079adf-7326-4580-b6e7-686339a04fef.mp4

**Comment by fawaz-ahmed** [comment]
@matziol thanks for bringing this up. I'll debug this.

**Comment by fawaz-ahmed** [comment]
@matziol fix released in ver 2.3.7

**Comment by matziol** [comment]
Working well, thank you!
